tests/conftest.py

```diff
@@ -16,7 +16,7 @@ def is_responsive(url):
         return False
 
 
-@pytest.fixture(scope="session", params=["lab", "base-with-services"])
+@pytest.fixture(scope="session", params=["lab", "base-with-services", "full-stack"])
```

Comment:
If you look at [stack/full-stack/Dockerfile](https://github.com/aiidalab/aiidalab-docker-stack/pull/372/files#diff-20a2c9d37a1aa2cd8ffb85dc4c0c8126e73ef093ae060c82563470a1dbdc8dac) you'll find I have re-install `erlang` even though it is already installed in the `base-with-services`. The reason is that we use the multiple stage and packages installed in the preliminary stage will not transfer to the final image. This test is will fail if `erlang` not installed explicitly in `full-stack`.

Reply:
Right, fair enough. I guess my question then is whether there is enough reason to test the "lab" image separately, or whether just testing `base-with-services` and `full-stack` is enough (which would also allow us to get rid of bunch of extra skipping logic in some tests. But that can be left for another PR.

Reply:
> whether just testing base-with-services and full-stack is enough

I am second on this. I didn't see the point to test `lab` and `full-stack`. Can you open an issue and assign it to me and you? Should be a quick fix I assume.

Reply:
I tried to remove the test of `base-with-services` in https://github.com/aiidalab/aiidalab-docker-stack/pull/385. The lab is worth testing because we want to be sure strating with docker-compose is working.